### DISCUSSION TITLE
Howto use Sold Services

### DISCUSSION THREAD

**Discussion by antcorro** [YetiForceCompany/YetiForceCRM#15570] [How to]
Hello.

I don't understand how to use the "Sold Services". Why I cannot edit a Sold Service after creating it?

I would like to manage this scenario:
I'm selling a Service that will be active for one year (like hosting or domain), So I need to track the start date and the expiration date, so that I can have an overview of expiring services and try to sell the renewal.

Do you have any suggestions on how to do this in YetiForce? 

Thank You

Regards
Antonio

**Comment by antcorro** [comment]
Hello,

Is there a way to edit again a Sold Service in status "Accepted"?

Is there a way to avoid the Sold Service to became not editable? 

Thank you

Regards 
Antonio

**Comment by mariuszkrzaczkowski** [comment]
> Is there a way to edit again a Sold Service in status "Accepted"?

Yes , you probably have an active one of the mechanisms that block the edit of the record

look at this  ​https://github.com/YetiForceCompany/YetiForceCRM/discussions/15503

**Reply by antcorro** [reply]
Thank you @mariuszkrzaczkowski ,
but I have already checked the workflow and the picklist lock status. There are no "block edition" workflows on Sold Services and no lock for "Accepted" value in Status picklist.
I also tried on a fresh installation YetiForce v6.3 and the default behavior is to lock the Sold Services in Accepted Status. This happens also in gitstable.yetiforce.com

Any suggestions?

Thank you

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/blob/b3cbd8185f05f784e4cb7099ca418915f395dd12/modules/OSSSoldServices/OSSSoldServices.php#L15

**Reply by antcorro** [reply]
ok Thank you. So this means that the only way is to modify the code of my instance?

**Reply by vovpff** [reply]
Yep
